package.json

```diff
@@ -92,9 +94,16 @@
     "react-is": "^18.2.0",
     "rollup-plugin-postcss": "^4.0.2",
     "size-limit": "^8.0.1",
+    "storybook": "^7.5.3",
     "tiny-invariant": "^1.2.0",
     "tsdx": "^0.14.1",
     "tslib": "^2.4.0",
-    "typescript": "^4.7.4"
+    "typescript": "^4.7.4",
+    "vite": "^4.5.0"
+  },
+  "resolutions": {
+    "string-width": "^4",
+    "jackspeak": "^2",
+    "strip-ansi": "^6"
```

Comment:
This gave me an error locally when trying to build after a clean install:
```
node:internal/modules/cjs/loader:1085
  throw err;
  ^

Error: Cannot find module 'strip-ansi'
Require stack:
- /Users/ealush/dev/emoji-picker-react/node_modules/ora/index.js
- /Users/ealush/dev/emoji-picker-react/node_modules/tsdx/dist/index.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1082:15)
    at Module._load (node:internal/modules/cjs/loader:928:27)
    at Module.require (node:internal/modules/cjs/loader:1149:19)
    at require (node:internal/modules/helpers:121:18)
    at Object.<anonymous> (/Users/ealush/dev/emoji-picker-react/node_modules/ora/index.js:7:19)
    at Module._compile (node:internal/modules/cjs/loader:1267:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1321:10)
    at Module.load (node:internal/modules/cjs/loader:1125:32)
    at Module._load (node:internal/modules/cjs/loader:965:12)
    at Module.require (node:internal/modules/cjs/loader:1149:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/ealush/dev/emoji-picker-react/node_modules/ora/index.js',
    '/Users/ealush/dev/emoji-picker-react/node_modules/tsdx/dist/index.js'
  ]
}

Node.js v20.0.0
```

Can you try running

```
git clean -xdf
```

And then reinstall to see if this replicates on your end as well?

Reply:
you totally right, this module is not used